app/views/static/landing_page.html.erb

```diff
@@ -0,0 +1,613 @@
+<%= render "shared/landingheader" %>
+<body>
+<%= render "shared/landingbodyheader" %>
```

Comment:
indent this and footer, plus any paragraph continuations

Reply:
fixed.